app/actions/publish.ts

```diff
@@ -14,7 +14,20 @@ export async function publish(
   teamID: string | undefined,
   accessToken: string | undefined,
 ) {
+  try {
+    const parsedUrl = new URL(url)
+    if (!parsedUrl.hostname.endsWith('.e2b.app')) {
+      throw new Error('URL must be on *.e2b.app domain')
+    }
```

Comment:
This should be outside the try block, otherwise you will suppress the message.

Reply:
I think it's fine to leave it without try catch, sorry, there's no much added value in custom error. It just makes the code more complicated